airbyte-integrations/connectors/source-hubspot/source_hubspot/api.py

```diff
@@ -436,6 +436,7 @@ class IncrementalStream(Stream, ABC):
 
     state_pk = "timestamp"
     limit = 1000
+    need_chunk = True
 
```

Comment:
Please add a comment here why and where this flag is used

Reply:
Updated.